assessments/projects/custodian/joint-review.md

```diff
@@ -0,0 +1,556 @@
+benefit of the user community.  That said, the project volunteers could use additional security volunteers to assist in ongoing reviews, security control 
+enhancements, and when needed to debug and fix security flaws.  As a utility tool, the review team feels the primary threat to the project is compromise of the code
+itself, and packaging and distribution of the tool or its dependencies - rather than an attack on individual c7n deployments. A secondary concern would the disabling
+or DoS of a deployment by an internal or external actor - thus giving the user a false sense of security when c7n is disabled. A tertiary concern is the
+provisioning of too permissive a policy within the cloud environment for c7n; this would make it an attrative target for lateral movement and other activities
+for an attacker who had gained access via some other means. Finally, comprehensive support for Kubernetes is not currently available; the review team would like to see more
+active support for Kubernetes and other CNCF projects as c7n matures.
+
+A “hands-on” penetration test was not performed, nor was a comprehensive code review, code fuzzing, or code audit.  The review team strongly recommends pursuing this as a 
+follow up with appropriate CNCF support and guidance.
+
+## Table of Contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Joint-review use](#joint-review-use)
+* [Intended use](#intended-use)
+* [Project design](#project-design)
+  * [Functions and features](#functions-and-features)
+    * [Security functions and features](#security-functions-and-features)
+* [Configuration and set-up](#configuration-and-set-up)
+* [Project compliance](#project-compliance)
+* [Security analysis](#security-analysis)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+  * [Closed security issues and vulnerabilities](#closed-security-issues-and-vulnerabilities)
+* [Roadmap](#roadmap)
+* [Appendix](#appendix)
+
+## Metadata
+
+| Project Home                    | https://github.com/cloud-custodian                                                                               |
+|---------------------------------|------------------------------------------------------------------------------------------------------------------|
+| Security Provider               | Yes, however many users implement Cloud Custodian for non security purposes, i.e., cost optimization, governance |
+| Languages                       | Python, YAML |
+| Incubation PR                   | https://github.com/cncf/toc/pull/480 |
+| SIG-Security Assessment Request | https://github.com/cncf/sig-security/issues/307 |
+| Self-Assessment Document        | https://docs.google.com/document/d/1s88ifDtFJzGX1O_ve6HM6_vbTkpsWDKWRDuxt9RHdcs/ |
```

Comment:
I believe this document has to be converted as well and checked into the PR

Reply:
Yep - it does, they should both go in the folder.

Reply:
Besides the self assessment doc, is there anything else that would need to be archived for posterity? (i.e. minutes/checkins, etc.)?

Reply:
do we need the project's approval for that?

Reply:
The self-assessment should be owned by the project and submitted as a PR to our repo for independent attestation by the security group or minimally have the markdown linked in these docs